Deprecation warning in compilation re: PyUnicode_GetSize

Probably doesn't matter too much, but compiling astropy (on Py3.7) gets (in particular) the following deprecation warning:
```
astropy/io/votable/src/tablewriter.c: In function ‘write_tabledata’:
astropy/io/votable/src/tablewriter.c:274:17: warning: ‘PyUnicode_GetSize’ is deprecated [-Wdeprecated-declarations]
                 str_len = PyUnicode_GetSize(str_val);
                 ^~~~~~~
In file included from ../../../miniconda3/include/python3.7m/Python.h:80,
                 from astropy/io/votable/src/tablewriter.c:12:
../../../miniconda3/include/python3.7m/unicodeobject.h:793:24: note: declared here
 PyAPI_FUNC(Py_ssize_t) PyUnicode_GetSize(
                        ^~~~~~~~~~~~~~~~~
```
(tablewriter.c is a handwritten C-extension, and PyUnicode_GetSize (https://docs.python.org/3/c-api/unicode.html#c.PyUnicode_GetSize) is deprecated since Py3.3 (looking at the sources).

--- a/astropy/io/votable/src/tablewriter.c
+++ b/astropy/io/votable/src/tablewriter.c
@@ -271,7 +271,7 @@ write_tabledata(PyObject* self, PyObject *args, PyObject *kwds)
                     goto exit;
                 }
 
-                str_len = PyUnicode_GetSize(str_val);
+                str_len = PyUnicode_GetLength(str_val);
                 if (str_len) {
                     if (_write_cstring(&buf, &buf_size, &x, "  <TD>", 6) ||
                         _write_string(&buf, &buf_size, &x, str_tmp, str_len) ||

--- a/astropy/utils/xml/src/iterparse.c
+++ b/astropy/utils/xml/src/iterparse.c
@@ -1184,7 +1184,7 @@ _escape_xml(PyObject* self, PyObject *args, const char** escapes)
             return NULL;
         }
 
-        input_len = PyUnicode_GetSize(input_coerce);
+        input_len = PyUnicode_GetLength(input_coerce);
 
         for (i = 0; i < input_len; ++i) {
             for (esc = escapes; ; esc += 2) {

